packages/css-library/src/stylesheets/uswds-typography.scss

```diff
@@ -23,16 +23,4 @@
 */
 p, a, ol, ul, li, dl, dt, dd, form, label, button {
   @include typeset('ui');
-}
-
-/**
-  USWDS paragraph style
-
-  p {
-    line-height: 1.5;
-    max-width: 68ex;
-    margin-bottom: 0;
-    margin-top: 0;
-  }
-*/
-@include usa-paragraph-style;
```

Comment:
The paragraph styles were causing issues with margin/padding and max-width that we are not prepared to address site wide. Removing this does not affect the normalization effort though.


```
p {
  line-height: 1.5;
  max-width: 68ex;
  margin-bottom: 0;
  margin-top: 0;
}
* + p {
  margin-top: 1em;
}
p + * {
  margin-top: 1em;
}
```